src/config.json

```diff
@@ -26,6 +26,18 @@
               }
           ]
         },
+        "notifications": {
+            "title": "Notifications (Beta)",
+            "permissions": [
+                {
+                    "method": "hasPermissions",
+                    "args": [["notifications:*:*"]]
+                },
+                {
+                    "method": "isBeta"
```

Comment:
@karelhala @ryelo :
Is possible to negate isBeta?

I would like to know if is possible to. hide policies settings when using stable.

Reply:
That's exactly what the `isBeta` does, it will be visible only for beta environments. Hidden on any stable ones.

Reply:
Sorry, I worded it the other way. I want to hide it in `beta` in the end.

Is still unclear if we want to do that, but in `beta` we want to replace our Policies email preferences with Notifications.

So we have Policies and Notifications and something like this is being discussed:

Beta:
 Only shows Notifications

Stable:
 Only shows Policies

Reply:
Oh, okay. There is no check for that thing, but you can write it you want and enhance this constant https://github.com/RedHatInsights/user-preferences-frontend/blob/master/src/Utilities/functions.js#L10